app/components/Home.tsx

```diff
@@ -159,9 +164,12 @@ export class Home extends React.Component<{}, AppProps> {
   }
 
   updatePreviewText = () => {
-    const preview = document.getElementById("preview");
-    if (preview) {
-      preview.innerHTML = MarkdownParser.render(this.state.text);
+    // Do not update preview text on Edit only mode to improve performance
+    if (!(this.state.mode === 0)) {
```

Comment:
Shouldn't be using `0` here directly. Use the named constant (or enums, when they are implemented).

Reply:
Also should be `!==`